packages/create-sitecore-jss/src/templates/react/package.json

```
@@ -0,0 +1,122 @@
+{
+  "name": "<%- appName %>",
+  "description": "Application utilizing Sitecore JavaScript Services and React (create-react-app).",
+  "version": "20.0.0-canary",
+  "private": true,
+  "config": {
+    "appName": "<%- appName %>",
+    "rootPlaceholders": [
+      "<%- appPrefix ? `${helper.getPascalCaseName(appName)}-` : '' %>jss-main"
+    ],
+    "sitecoreDistPath": "/dist/<%- appName %>",
+    "sitecoreConfigPath": "/App_Config/Include/zzz",
+    "graphQLEndpointPath": "/sitecore/api/graph/edge",
+    "language": "en",
+    "tunnelUrl": "http://jss.ngrok.io"
+  },
+  "engines": {
+    "node": ">= 12",
+    "npm": ">=6"
+  },
+  "author": {
+    "name": "Sitecore Corporation",
+    "url": "https://jss.sitecore.com"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/sitecore/jss.git"
+  },
+  "bugs": {
+    "url": "https://github.com/sitecore/jss/issues"
+  },
+  "license": "Apache-2.0",
+  "dependencies": {
+    "@sitecore-jss/sitecore-jss-react": "^20.0.0-canary.77",
```

remove version number
```suggestion
    "@sitecore-jss/sitecore-jss-react": "^20.0.0-canary",
```